src/services/pps.ts

```diff
@@ -213,6 +224,29 @@ const pps = ({
       return streamToObjectArray<PipelineInfo, PipelineInfo.AsObject>(stream);
     },
 
+    deletePipeline: ({
+      pipeline,
+      keepRepo = false,
```

Comment:
We'll want to double check we have the correct defaults here before it gets implemented in Console. This makes sense to me, but I'm not sure what an average user would expect.

Reply:
yeah I based it off of pachctl's default behavior, looks like the python client also defaults it to false? https://github.com/pachyderm/python-pachyderm/blob/1cb4e02a73ab8dafc08837ba25221e42da04dfee/src/python_pachyderm/proto/v2/pps/pps_pb2.py